api/src/main/java/org/openmrs/module/eptsreports/EptsReportsActivator.java

```diff
@@ -58,7 +59,7 @@ public void started() {
 		try {
 			reportsInitializer.initializeReports();
 		}
-		catch (IllegalArgumentException e) {
+		catch (MetadataLookupException e) {
```

Comment:
@ynurmahomed this looks good :+1: 

Only suggestion I have is to also add a catch for a generic exception to print a general message in the event of any other kind of error.